13/realnitinworks/directors.py

```diff
@@ -0,0 +1,118 @@
+from collections import namedtuple, Counter, defaultdict
+import csv
+import statistics
+
+MOVIES_DATASET = "movie_metadata.csv"
+FROM_YEAR = "1960"
+MIN_MOVIES = 4
+TOP_N = 20
+
+Movie = namedtuple('Movie', 'director title year rating')
+
+
+def convert_csv_to_dict(data=MOVIES_DATASET):
+    """ Yield Movie(director, title, year, rating) record from input csv file """
+    with open(data) as csvfile:
+        for movie in csv.DictReader(csvfile):
+            yield Movie(
+                director=movie['director_name'],
+                title=movie['movie_title'].strip(),
+                year=movie['title_year'],
+                rating=movie['imdb_score']
+            )
+
+
+def from_year(movies, year=FROM_YEAR):
+    """ Filter movies based on year"""
+    for movie in movies:
+        if not year:
+            continue
+        if movie.year >= year:
```

Comment:
could combine? `if year and movie.year >= year`

Reply:
Oh! Why do I miss these 

> could combine? `if year and movie.year >= year`

Ohh! Why do I miss these . Thanks Bob